Dockerfile

```diff
@@ -35,10 +35,6 @@ RUN apt-get update \
     && apt-get install -y --no-install-recommends postgresql-client-$PG_MAJOR \
     && rm -rf /var/lib/apt/lists/*
 
-RUN mkdir /code
-WORKDIR /code
-
-RUN mkdir /code/listenbrainz
 WORKDIR /code/listenbrainz
```

Comment:
Yeah, WORKDIR will run it as the root user. But I also found https://github.com/moby/moby/issues/36677 just now which suggests to create the directory manually before WORKDIR. In our case, its all running as root (according to my understanding) so maybe its fine? What do you think?

Reply:
Interesting comment. I guess if we were adding files as a non-privileged user then this could be a problem, though everything is just as root now so there's no difference.
Perhaps as docker suggests going the mkdir route we could do that for consistency, and also so that we don't run into this problem if we ever change the user permissions.
To simplify this code we could just have a single `RUN mkdir -p /code/listenbrainz /static` in one place.